drivers/adc/adc_nrfx_saadc.c

```diff
@@ -374,6 +390,8 @@ static int init_saadc(struct device *dev)
 {
 	nrf_saadc_event_clear(NRF_SAADC_EVENT_END);
 	nrf_saadc_int_enable(NRF_SAADC_INT_END);
+	nrf_saadc_event_clear(NRF_SAADC_EVENT_CALIBRATEDONE);
+	nrf_saadc_int_enable(NRF_SAADC_INT_CALIBRATEDONE);
```

Comment:
This is why I don't like nrfx hal: if I could just do:
```
NRF_SAADC->INTEN = (SAADC_INTENSET_CALIBRATEDONE_Set << SAADC_INTENSET_CALIBRATEDONE_Pos)
                 | (SAADC_INTENSET_END_Set << SAADC_INTENSET_END_Pos);
```
I'd know what was going to happen, rather than having to dig into the HAL source to see what `nrfc_saadc_int_enable` does with its arguments.

Done.